JSTask5.js

```diff
@@ -0,0 +1,169 @@
+// Task 1
+// Remove duplicates of the array and return an array of only unique elements.
+
+const array = [1, 2, 3, 5, 1, 5, 9, 1, 2, 8];
+let dup = [...new Set(array)];
+console.log(dup);
+
+
+// Task 2
+/* Find the intersection of two arrays. (An intersection would be the common elements that exist within both arrays).
+The elements should be unique! */
+
+const firstArray = [2, 2, 4, 1];
+const secondArray = [1, 2, 0, 2];
+let intersection = firstArray.filter(value=>secondArray.includes(value));
```

Comment:
Output of the code given here is [2, 2, 1]. Common elements shouldn't have duplicates